src/views/GovernanceReviewTeam/index.scss

```diff
@@ -0,0 +1,29 @@
+.easi-grt {
+  &__request-summary {
+    background-color: #0071bc;
+    color: #fff;
+  }
+
+  &__request-info {
```

Comment:
Instead of making our own classes here, could we lean on the grid layout baked into USWDS?

Reply:
I thought about this too. I opted to create our own class because:
1. We'd be using 3 USWDS utility class names and we're straddling the line between using too many utility classes.
2. Since I want to target `dt` and `dd` inside of the element, I need a class name to target them, so I figured to just include the styles rather than use utility classes.

Reply:
I hear ya on a lot of utility classes... One thing I've done in the past is set up a variable for lots of combined classes we are applying. I think either approach is ok here, tho, so approving.